easybuild/easyconfigs/d/DeePDM-kit/DeePDM-kit-3.0.1-foss-2023a-CUDA-12.1.1-with-LAMMPS-plugin.eb

```
@@ -0,0 +1,115 @@
+easyblock = 'PythonBundle'
+
+name = 'DeePDM-kit'
```

Small typo with huge impact. @pavelToman can look into updating the pr and renaming the easyconfig file
```suggestion
name = 'DeePDM-kit'
```

---

I hope this is already fixed?

---

Ah, there are 2 PRs... so it is fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/22217, gonna fix it either here!